engine/src/main/coffee/engine/plot/plotmanager.coffee

```diff
@@ -196,6 +213,18 @@ module.exports = class PlotManager
     pipeline(values, forEach(f))(@_plotMap)
     return
 
+  # (String) => Plot
+  _lookupPlot: (name) ->
+
+    upperName = name.toUpperCase()
+    plot      = @_plotMap[upperName]
+
+    if plot?
+      plot
+    else
+      f = ([k, v]) -> k.match("__HNW_ROLE_[^_]+_#{upperName}")?
```

Comment:
I'm definitely okay with this as it's simple and will be pretty obvious to anyone looking at it what it's doing.  But I wonder if there would be some clean way to enable this without hard-coding the knowledge of HNW.  Maybe if we ever get a second similar use-case.